### DISCUSSION TITLE
Welcome to kinx Discussions!

### DISCUSSION THREAD

**Discussion by Kray-G** [Kray-G/kinx#149] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by htsign** [comment]
Stars has increased, so I try to post in English as much as possible😇

**Reply by Kray-G** [reply]
Thanks for the comment!
I am very appreciative of increasing starts.
I might not be able to meet expectations of a lot of people, but I would like to keep doing a small thing.

I did open the discussion tab, so feel free to post anything if you have any theme for some future implementation, specification, features, and so on.

**Comment by htsign** [comment]
I found out just a while ago, This language picked up in Hacker News.
https://news.ycombinator.com/item?id=25787716
